website/registrations/models.py

```diff
@@ -364,7 +364,7 @@ class Renewal(Entry):
 
     @property
     def payment_payer(self):
-        return self.member
+        return PaymentUser.objects.get(pk=self.member.pk)
```

Comment:
Why does this need to be a PaymentUser exactly? The place where this property is used could convert it, then you don't need to import the model here